microsoft-edge/devtools-guide-chromium/experimental-features/copilot-explain.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Explain errors and source code in Copilot in Edge
+description: Discover new experimental features to explain Console errors and source code by using Copilot in Edge
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 12/07/2023
+---
+# Explain errors and source code in Copilot in Edge
+
+The **Console** and **Sources** tools in Microsoft Edge DevTools integrate with Copilot in Microsoft Edge, as an experiment, to help you understand complex errors and source code.
+
+When inspecting a webpage with DevTools, you often see errors and warnings in the **Console** tool. These errors and warnings can sometimes be difficult to understand and fix. By using the **Explain this error** feature, you get more information about the error or warning in Copilot in Edge and learn how to fix the problem.
```

Comment:
a screenshot of either or both the Console / sources with a red box around the icon would be helpful here. I also suggest keeping part or all of the image ATF.

Reply:
copied two images from below into the new h4 for Errors & h4 for Source code